addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/scanrules/PassiveScriptScanRule.java

```diff
@@ -79,15 +79,21 @@ public PluginPassiveScanner copy() {
 
     @Override
     public AlertBuilder newAlert() {
-        return super.newAlert()
-                .setRisk(metadata.getRisk().getValue())
-                .setConfidence(metadata.getConfidence().getValue())
-                .setDescription(metadata.getDescription())
-                .setSolution(metadata.getSolution())
-                .setCweId(metadata.getCweId())
-                .setWascId(metadata.getWascId())
-                .setReference(String.join("\n", metadata.getReferences()))
-                .setOtherInfo(metadata.getOtherInfo());
+        var alertBuilder =
+                super.newAlert()
+                        .setRisk(metadata.getRisk().getValue())
+                        .setConfidence(metadata.getConfidence().getValue())
+                        .setDescription(metadata.getDescription())
+                        .setSolution(metadata.getSolution())
+                        .setCweId(metadata.getCweId())
+                        .setWascId(metadata.getWascId())
+                        .setOtherInfo(metadata.getOtherInfo());
+        if (metadata.getReferences() != null && !metadata.getReferences().isEmpty()) {
+            alertBuilder.setReference(String.join("\n", metadata.getReferences()));
+        } else {
+            alertBuilder.setReference("N/A");
```

Comment:
i18n this string?

Reply:
For both usages

Reply:
Should be an empty string, though in this case it does not need to set any string.

Reply:
My bad, we were supposed to remove "N/A" instead of adding it. I'll replace this with an empty string.

Reply:
Okay